src/Symfony/Component/Security/Core/Authorization/AccessDecisionManagerInterface.php

```diff
@@ -26,7 +26,7 @@ interface AccessDecisionManagerInterface
      * @param array  $attributes An array of attributes associated with the method being invoked
      * @param object $object     The object to secure
      *
-     * @return bool true if the access is granted, false otherwise
+     * @return bool|AccessDecision true if the access is granted, false otherwise
```

Comment:
What about adding a new decideWithDecision() method?

(Or another name)

Or the need is to have one and only one method/entrypoint?

This return feels weird IMO

Reply:
We need here only one method called decide in this contract imo. `bool` in the return type is kept for BC.